Source/GRChomboCore/BoundaryConditions.cpp

```diff
@@ -275,18 +269,15 @@ void BoundaryConditions::write_mixed_conditions(int idir,
 {
     // check all the vars have been assigned a BC - this should always be the
     // case because of how the params are assigned
-    CH_assert(a_params.mixed_bc_sommerfeld_vars.size() +
-                  a_params.mixed_bc_extrapolating_vars.size() ==
-              NUM_VARS);
+    CH_assert(a_params.mixed_bc_vars_map.size() == NUM_VARS)
```

Comment:
missing a semicolon here?

Reply:
Now I'm confused why the ScalarField example I compiled (with `OPT` not `HIGH`) didn't complain.

Reply:
fixed